dogu.json

```diff
@@ -18,6 +18,11 @@
     "Description": "URL of SCM-Manager Plugin Center",
     "Optional": true
   },
+  {
+    "Name": "scm_plugins_default",
```

Comment:
Why not something simple? Such as `default_plugins`?

Reply:
I thought it would be more explicit since we also have default plugins for sonar and jenkins.

Reply:
This `configuration` is in the **scm** dogu.json and is stored at `/config/scm/...`. I think this explicit enough.

Reply:
Ok, i've done a short check how the other configurations for default plugins look like:

* sonar uses [sonar.plugins.default](https://github.com/cloudogu/sonar/blob/develop/dogu.json#L57)
* jenkins uses [additional.plugins](https://github.com/cloudogu/jenkins/blob/develop/dogu.json#L46)

Both of them use dot notation, so we should do the same. I don't like `sonar.plugins.default`, because of the duplicated dogu name, but i like `additional.plugins`.

I think we should go with **additional.plugins**. It is analog to jenkins and it is more precise than default plugins, because this setting configures additional plugins and does not set the default plugins.